home/views.py

```diff
@@ -5,3 +5,8 @@ def index(request):
     """ A view to return the index page """
 
     return render(request, "home/index.html")
+
```

Comment:
We are following PEP8 standard where you would want 2 line breaks between functions or classes. Could you please add another line break here? If you are interested, you can read more here: https://www.python.org/dev/peps/pep-0008/